### PR TITLE
Ccspv2 report and report enhancements

### PR DESCRIPTION
**Pull request by Ladas**

### CODE REVIEW

File: metrics_utility/automation_controller_billing/collectors.py [hunk 1]
```diff
@@ -238,3 +244,62 @@ def job_host_summary_table(since, full_path, until, **kwargs):
...
+            {tbl}.play,
+            {tbl}.task,
+            {tbl}.role,
+            {tbl}.job_id as job_remote_id,
+            {tbl}.host_id as host_remote_id,
+            {tbl}.host_name
+
+        FROM {tbl}
+        JOIN job_scope ON job_scope.job_created = {tbl}.job_created AND job_scope.job_id={tbl}.job_id AND job_scope.host_name={tbl}.host_name
+        WHERE {tbl}.event IN ('runner_on_ok',
```
Comment (slemrmartin) [inline metrics_utility/automation_controller_billing/collectors.py:293]: is it indexed?

File: README.md [hunk 2]
```diff
@@ -57,6 +57,35 @@ export METRICS_UTILITY_REPORT_RHN_LOGIN="test_login"
...
+
+
+Example with local directory storage CCSPv2 type:
+```
+# Set needed ENV VARs for data gathering
+export METRICS_UTILITY_SHIP_TARGET=directory
+export METRICS_UTILITY_SHIP_PATH=/awx_devel/awx-dev/metrics-utility/shipped_data/billing
+
+# Set extra ENV VARs for report generation purposes
+export METRICS_UTILITY_REPORT_TYPE=CCSPv2
```
Comment (slemrmartin) [inline README.md:74]: following 5 lines are duplicates

File: metrics_utility/automation_controller_billing/collectors.py (hunk 1 above)
Comment (Ladas) [reply metrics_utility/automation_controller_billing/collectors.py:293]: yeah events are partitioned on job_created and there is index on job_id, so it does the right partition pruning and index scan

File: README.md (hunk 2 above)
Comment (Ladas) [reply README.md:74]: yeah, but it makes sense to document it per type, regardless of which env vars are reused for each type of report

Comment (slemrmartin) [reply README.md:74]: ah, I didn't realize this is readme, not a real export, so nevermind :D

**Pull request merged**